<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Профсоюз.Льготы</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(240, 240, 240)</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(254, 127, 94);
color: 'white'</string>
   </property>
   <property name="text">
    <string>     Профсоюз.Льготы</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>510</y>
     <width>421</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: 'white';
background: 'white';
border: 2px solid 'black'</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="home_btn">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>520</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: 'white';
border: 1px;
color: 'white'</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>home.PNG</normaloff>home.PNG</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="ur_help_btn">
   <property name="geometry">
    <rect>
     <x>286</x>
     <y>520</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px;
background: 'white'</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>help.png</normaloff>help.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="qr_button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>470</y>
     <width>100</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px;
background: 'white'</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>qr.png</normaloff>qr.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>130</width>
     <height>121</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="profile_btn">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>75</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(240, 240, 240);
border: 1px</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>profile.png</normaloff>profile.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="hello_lab">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(70, 70, 70)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QScrollArea" name="lgots_area">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>361</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(240, 240, 240);
border: 1px solid rgb(254, 127, 94);
</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>359</width>
      <height>279</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(70, 70, 70)</string>
   </property>
   <property name="text">
    <string>Ваши партнеры и льготы:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
